src/main/java/com/liferay/faces/test/selenium/IntegrationTesterBase.java

```diff
@@ -127,6 +127,6 @@ protected final void signIn(Browser browser, String signInURL, String loginXpath
 		WebElement loginElement = browser.findElementByXpath(loginXpath);
 		browser.click(signInButtonXpath);
 		browser.waitUntil(ExpectedConditions.stalenessOf(loginElement));
-		browser.waitForElementVisible("//body");
+		browser.waitForElementDisplayed("//body");
```

Comment:
@stiemannkj1 I really like the change from "visible" to "displayed" in all cases, except for this line that has xpath targeting the `<body>` element which is a non-rendered thing. Please let me know if you agree/disagree. Thanks.